### PR TITLE
Use standard login form for /admin/login/

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
This is a bit different than the approach we talked about. I felt weird about letting users access _/admin/_ if they weren't able to do anything there.

This isn't ideal either, but it shows a _Permission Denied_ notice if the user tried to access a page they don't have permission to view.

File: distributionviewer/core/templates/distributionviewer/login.html [hunk 1]
```diff
@@ -7,6 +7,14 @@
     <meta name="google-signin-client_id" content="{{ google_clientid }}">
     <title>Firefox Distribution Viewer</title>
     <link rel="stylesheet" type="text/css" href="{% static 'css/bundle.css' %}">
+    {% if user.is_authenticated %}
```
Comment (robhudson) [inline distributionviewer/core/templates/distributionviewer/login.html:10]: If you wanted to go one step further you could do `if user.is_authenticated and not user.is_staff`. I think that's more clear of the intention also.

**Comment by robhudson** [comment]
Aside from the one comment I approve of this message. 👍

File: distributionviewer/core/templates/distributionviewer/login.html (hunk 1 above)
Comment (openjck) [reply distributionviewer/core/templates/distributionviewer/login.html:10]: I was wondering about that. We could definitely special-case this for users who aren't able to access the _/admin/_ page. On the other hand, by checking only whether the user is authenticated, we can infer that they lacked permission to the intended page for any reason.

I used [this permission error message](https://github.com/django/django/blob/master/django/contrib/admin/templates/admin/login.html#L35-L42) as inspiration.

Comment (robhudson) [reply distributionviewer/core/templates/distributionviewer/login.html:10]: Works for me.

**Pull request merged**